app/src/test/java/org/simple/clinic/summary/addphone/AddPhoneNumberDialogControllerTest.kt

```diff
@@ -118,11 +112,14 @@ class AddPhoneNumberDialogControllerTest {
   @Suppress("unused")
   private fun `validation errors`() = listOf(Blank, LengthTooShort(6), LengthTooLong(12))
 
-  private fun setupController() {
+  private fun setupController(newNumber: String) {
     val controller = AddPhoneNumberDialogController(repository, validator, FakeUuidGenerator.fixed(generatedPhoneUuid))
 
     controllerSubscription = uiEvents
         .compose(controller)
         .subscribe { uiChange -> uiChange(dialog) }
+
+    uiEvents.onNext(AddPhoneNumberDialogCreated(patientUuid))
```

Comment:
One more thing I see that is missing in this PR is changing the controller to accept the parameters via assisted injection instead of a custom `ScreenCreated` event (See step 7 of [`Stabilize`](https://github.com/simpledotorg/simple-android/blob/master/doc/mobius/migrating-to-mobius.md#steps)).

Was this missed or we going to do it in another PR?

Reply:
Ah right, I missed it. Will fix it